pkg/controller/bootstrap/bootstrap.go

```diff
@@ -183,6 +188,14 @@ func (b *Bootstrap) Run(destDir string) error {
 		configs = append(configs, kconfigs...)
 	}
 
+	if infraConfig.Status.CPUPartitioning != apicfgv1.CPUPartitioningNone {
```

Comment:
Discussed on slack, but to clarify, if this only applies to bootstrap, if the MC does get generated, the install will fail due to a mismatch of configs generated by bootstrap and regular, causing a "can't find rendered-master-xxx" when the install happens. Basically https://github.com/openshift/machine-config-operator/issues/2114

Reply:
Hey @yuqi-zhang , was looking through the code a bit, trying to understand this a little better. Does this mean that it's not possible to have a MC generated and maintain by MCO during bootstrap?

Reply:
No, it is possible. You must have both the bootstrap and the "in-cluster" flow generate the same configs.

As an example, see this: https://github.com/openshift/machine-config-operator/pull/3015

which adds both for nodes.config.openshift.io objects